src/periodograms.jl

```diff
@@ -162,17 +162,18 @@ function fft2oneortwosided!{T}(out::Array{Complex{T}}, s_fft::Vector{Complex{T}}
     out
 end
 
-# Evaluate a window function at n points, returning both the window
-# (or nothing if no window) and the squared L2 norm of the window
-compute_window(::Nothing, n::Int) = (nothing, n)
-function compute_window(window::Function, n::Int)
-    win = window(n)::Vector{Float64}
-    norm2 = sumabs2(win)
-    (win, norm2)
-end
-function compute_window(window::AbstractVector, n::Int)
-    length(window) == n || error("length of window must match input")
-    (window, sumabs2(window))
+# Calculate sum of abs2
+# Remove this once we drop support for Julia 0.2
+if isdefined(Base, :sumabs2)
+    sumabs2(x) = Base.sumabs2(x)
+else
+    function sumabs2(s)
+        x = zero(eltype(s))
+        for i = 1:length(s)
+            @inbounds x += abs2(s[i])
+        end
+        x
+    end
 end
```

Comment:
I removed this in 5c016833c48cd622f949ea458a4a886c9f590741 since we now require 0.3+ and so we don't need it. It looks like it came back from the dead during the rebase? :dizzy_face: